sql/migrations/20211122153748_world.sql

```diff
@@ -0,0 +1,136 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20211122153748');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20211122153748');
+-- Add your query below.
+
+
+-- Creature 915
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 4659 AND `id` = 1;
+
+-- Creature 916
+UPDATE `gossip_menu` SET `text_id` = 4834 WHERE `entry` = 4676 AND `text_id` = 4835;
+UPDATE `gossip_menu_option` SET `option_text` = 'I\'m lookin\' for rogue trainin\'.', `option_broadcast_text` = 7490 WHERE `menu_id` = 4676 AND `id` = 0;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 4676 AND `id` = 1;
+
+-- Creature 917
+UPDATE `gossip_menu` SET `text_id` = 878 WHERE `entry` = 381 AND `text_id` = 4795;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 381 AND `id` = 1;
+
+-- Creature 918
+UPDATE `gossip_menu` SET `text_id` = 5993 WHERE `entry` = 4502 AND `text_id` = 5996;
+
+-- Creature 1411
+DELETE FROM `gossip_menu` WHERE `entry` = 4658 AND `text_id` = 5996;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 4658 AND `id` = 4;
+
+-- Creature 1234
+UPDATE `gossip_menu` SET `condition_id` = 100 WHERE `entry` = 410 AND `text_id` = 907;
+UPDATE `gossip_menu` SET `condition_id` = 101 WHERE `entry` = 410 AND `text_id` = 4797;
+DELETE FROM `gossip_menu` WHERE `entry` = 410 AND `text_id` = 4795;
+UPDATE `gossip_menu_option` SET `option_broadcast_text` = 2929, `option_text` = 'Good day to you, Hogral! I seek training.' WHERE `menu_id` = 410 AND `id` = 0;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 410 AND `id` = 1;
+
+-- Creature 2122
+-- Creature 2130
+DELETE FROM `gossip_menu` WHERE `entry` = 85 AND `text_id` = 5996;
+DELETE FROM `gossip_menu_option` WHERE `menu_id`= 85 AND `id` = 4;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 85 AND `id` = 1;
+
+-- Creature 3155
+-- Creature 3170
+DELETE FROM `gossip_menu` WHERE `entry` = 141 AND `text_id` = 5996;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 141 AND `id` = 4;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 141 AND `id` = 1;
+
+-- Creature 3328
+UPDATE `gossip_menu` SET `text_id` = 5993 WHERE `entry` = 4513 AND `text_id` = 5996;
+
+-- Creature 3401
+UPDATE `gossip_menu` SET `text_id` = 5993 WHERE `entry` = 521 AND `text_id` = 5996;
+UPDATE `gossip_menu_option` SET `option_broadcast_text` = 3143, `option_text` = 'I require training, sir!' WHERE `menu_id` = 521 AND `id` = 0;
+
+-- Creature 3594
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 4690 AND `id` = 1;
+
+-- Creature 3599
+UPDATE `gossip_menu` SET `text_id` = 934 WHERE `entry` = 436 AND `text_id` = 4794;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 436 AND `id` = 1;
+
+-- Creature 4163
+UPDATE `gossip_menu` SET `text_id` = 4795 WHERE `entry` = 4576 AND `text_id` = 4794;

```

Comment:
You are replacing text id 4794 with 4795, but both of those are part of gossip menu 4576 according to sniff.
![Untitled](https://user-images.githubusercontent.com/35845488/148131617-c4297cb4-1f60-4fe9-b240-52126cfe1e6e.jpg)

Reply:
text 4795 seems to be her default rogue greeting, not sure how 4794 is activated.